x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/rule_creation/create_rules.ts

```diff
@@ -444,272 +408,279 @@ export default ({ getService }: FtrProviderContext) => {
           await deleteUserAndRole(getService, role);
         });
 
-        it('should NOT be able to create a rule', async () => {
+        it('does NOT create a rule', async () => {
```

Comment:
I think the original message makes more sense: the `t1_analyst` is NOT able to create a rule, rather than "does not create a rule"